package.json

```diff
@@ -68,6 +68,7 @@
     "build:docs": "cp css/emoji-mart.css docs && webpack --config ./docs/webpack.config.js",
     "build": "npm run clean && npm run build:dist",
     "watch": "BABEL_ENV=cjs babel src --watch --out-dir dist --copy-files",
+    "watch:es": "babel src --watch --out-dir dist-es --copy-files",
```

Comment:
Was committing this intentional? If so, we should pull it into a dedicated commit, with a message explaining why this is useful.

Reply:
Not intentional. Removing it from this PR. Maybe we should make a separate PR for this?